WordPress/Classes/Utility/Ratings/AppRatingsUtility.swift

```diff
@@ -18,6 +18,16 @@ class AppRatingUtility: NSObject {
     ///
     var appReviewUrl: URL = Constants.defaultAppReviewURL
 
+    /// Don't prompt for reviews for internal builds
+    /// http://stackoverflow.com/questions/27297435/detect-if-ios-app-is-downloaded-from-apples-testflight
+    ///
+    var isInternalBuild: Bool = {
+        guard let path = Bundle.main.appStoreReceiptURL?.path else {
```

Comment:
I think this should be moved to `FeatureFlag.swift`, where we could add a `ratingsPrompt` flag, or at least use `build(_)` to check if it's an internal build. I don't think `Build` can differentiate TestFlight betas right now, so we might have to add this there.